transformer_engine/tensorflow/recipe.py

```diff
@@ -0,0 +1,127 @@
+"""This module provides predefined FP8 recipes."""
```

Comment:
Why is this file necessary? Looks like it's *almost* copied from `transformer_engine/common/recipe.py`?

Reply:
Yes, you are right. Fixed by reusing the existing `recipe.py`. PTAL.